services/QuillLMS/engines/evidence/app/services/evidence/gen_ai/evaluation/verbose/checker.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+module Evidence
+  module GenAI
+    module Evaluation
+      module Verbose
+        class Checker < Evaluation::BaseChecker
+          MAX_SIZE = 300
+          MAX_SENTENCES = 4
+
+          def run
+            return suboptimal if too_long?
+
+            optimal
+          end
+
+          private def too_long?
+            return true if feedback_sentence_count > MAX_SENTENCES
+
+            feedback_sentence_count == MAX_SENTENCES &&
+            feedback.size > MAX_SIZE
```

Comment:
This line looks weird since the line is not indented.

Reply:
Yeah, I'll edit. The linter un-indented it.